src/base-element.js

```diff
@@ -802,6 +806,16 @@ export class BaseElement {
   }
 
   /**
+   * Called when an attribute's value changes.
+   * @param {!string} unusedName
+   * @param {?string} unusedOldValue
```

Comment:
Please explain in docs the meaning of `null` values. `newValue == null` means the attribute was removed?